acvm-repo/brillig_vm/src/lib.rs

```diff
@@ -228,7 +258,209 @@ impl<'a, F: AcirField, B: BlackBoxFunctionSolver<F>> VM<'a, F, B> {
 
         self.process_opcode_internal()
     }
+    fn trace_branching(&mut self, destination: usize) {
+        if self.fuzzing_active {
```

Comment:
We could do:

```rust
if !self.fuzzing_active {
  return;
}
```

in all of these to return the indentation level.

Reply:
Done